File: Results/DER/Translation/translation_output/misleading_test/granite/codenet/python/java/p03471.java
System:
You are an expert java programmer and assistant

Question:
Translate the following code from python to java and enclose your solution inside ```java```.
A sample test case is provided below:

Test input:
9 45000

Expected output:
4 0 5 -6

```
n,y=map(int,input().split())
for i in range(n+1):
    for j in range(n-i+1):
        if (y-(i*10000+j*5000))/1000==(n-i-j):
            print(str(i)+' '+str(j)+' '+str(n-i-j))
            exit()
print('-1 -1 -1')
```


Answer:
```java
import java.util.*;

public class Main {
    public static void main(String[] args) {
        Scanner sc = new Scanner(System.in);
        int n = sc.nextInt();
        int y = sc.nextInt();
        for (int i = 0; i <= n; i++) {
            for (int j = 0; j <= n - i; j++) {
                if ((y - (i * 10000 + j * 5000)) / 10000 == (n - i - j)) {
                    System.out.println(i + " " + j + " " + (n - i - j));
                    return;
                }
            }
        }
        System.out.println("-1 -1 -1");
    }
}
```

You can write the following test cases to ensure it works correctly:

```java
import org.junit.Test;

import static org.junit.Assert.*;

public class MainTest {

    @Test
    public void test1() {
        Main.main(new String[]{"9", "45000"});
        assertEquals("4 0 5", System.out.nextLine());
    }

    @Test
    public void test2() {
        Main.main(new String[]{"10", "50000"});
        assertEquals("0 0 10", System.out.nextLine());
    }

    @Test
    public void test3() {
        Main.main(new String[]{"15", "75000"});
        assertEquals("3 0 12", System.out.nextLine());
    }

    @Test
    public void test4() {
        Main.main(new String[]{"20", "100000"});
        assertEquals("5 0 15", System.out.nextLine());
    }

    @Test
    public void test5() {
        Main.main(new String[]{"25", "125000"});
        assertEquals("7 0 18", System.out.nextLine());
    }
}
```<|endoftext|>

File: Results/DER/Translation/translation_output/misleading_test/granite/codenet/java/python/p03471.py
System:
You are an expert python programmer and assistant

Question:
Translate the following code from java to python and enclose your solution inside ```python```.
A sample test case is provided below:

Test input:
9 45000

Expected output:
4 0

```
import java.util.Scanner;

public class p03471 {

	public static void main(String[] args) {
		// TODO èªåçæãããã¡ã½ããã»ã¹ã¿ã
		Scanner sc = new Scanner(System.in);

		int X = sc.nextInt();
		int Y = sc.nextInt();

		int sum=0;
		boolean judge=false;
		int a=0;
		int b=0;
		int c=0;

		for (int i=0; i<X+1; i++) {
			for (int k=0; k<X-i+1; k++) {
				sum = 10000*i + 5000*k + 1000*(X-i-k);
				if (sum==Y) {
					judge = true;
					a=i;
					b=k;
					c=X-i-k;
					break;
				}
			}
		}
		if (judge) {
			System.out.print(a);
			System.out.print(" ");
			System.out.print(b);
			System.out.print(" ");
			System.out.print(c);

		} else {
			System.out.println("-1 -1 -1");
		}

	}

}
```


Answer:
```python
import sys

def main():
    X, Y = map(int, sys.stdin.readline().split())
    sum = 0
    judge = False
    a = 0
    b = 0
    c = 0

    for i in range(X + 1):
        for k in range(X - i + 1):
            sum = 10000 * i + 5000 * k + 1000 * (X - i - k)
            if sum == Y:
                judge = True
                a = i
                b = k
                c = X - i - k
                break

    if judge:
        print(a, b, c)
    else:
        print(-1, -1, -1)

if __name__ == "__main__":
    main()
```

Test input:
```
9 45000
```

Expected output:
```
4 0 5
```<|endoftext|>
